bochs/bx_debug/dbg_main.cc

```diff
@@ -2283,6 +2283,55 @@ void bx_dbg_lyt(void)
   bx_nest_infile(layoutpath);
 }
 
+void bx_dbg_set_magic_bp_mask(Bit8u mask)
+{
+  bx_dbg.magic_break = mask;
+  dbg_printf("new magic breakpoint mask created: 0x%x\n", mask);
+}
+
+Bit8u bx_dbg_get_magic_bp_mask_from_str(const char *str)
+{
+  Bit8u new_mask = 0;
+
+  if (NULL == str)
+    {
```

Comment:
this is wrong way
the NULL == str was the only with correct indentation, all the rest needed update
now all need update